space2stats_api/src/space2stats/api/app.py

```diff
@@ -79,11 +203,20 @@ def get_aggregate(body: AggregateRequest, table: StatsTable = Depends(stats_tabl
 
     @app.get("/fields", response_model=List[str])
     def fields(table: StatsTable = Depends(stats_table)):
+        """Fields available in the statistics table"""
         return table.fields()
 
+    @app.get("/metadata")
+    def metadata_redirect():
+        """Redirect to project STAC Browser."""
+        return RedirectResponse(
+            "https://radiantearth.github.io/stac-browser/#/external/raw.githubusercontent.com/worldbank/DECAT_Space2Stats/refs/heads/main/space2stats_api/src/space2stats_ingest/METADATA/stac/catalog.json"
+        )
+
     @app.get("/")
-    def read_root():
-        return {"message": "Welcome to Space2Stats!"}
+    def docs_redirect():
+        """Redirect to project documentation."""
+        return RedirectResponse("https://worldbank.github.io/DECAT_Space2Stats")
```

Comment:
The top-level catalog enables navigating to the collection and underlying items.